<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataProcessingConfigurationForm</class>
 <widget class="QWidget" name="dataProcessingConfigurationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>430</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wavelet Denoising Configuration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wavelet:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Decomposition Level:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="waveletStrLineEd">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="decomposeLvlSpinBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Supported values:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>db1 .. db15</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>haar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>sym2 .. sym10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>coif1 .. coif5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior1.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior1.3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior1.5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior2.2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior2.4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior2.6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior2.8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior3.3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior3.5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior3.7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior3.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior3.9</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior6.8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior5.5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>bior4.4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
